### PR TITLE
Add ubuntu.pem and include trust store update script

### PR DESCRIPTION
**Pull request by anniephan**
ubuntu.pem is created by downloading ca-certificate package from ubuntu's archives. The current package downloaded is from ubuntu xenial (http://packages.ubuntu.com/source/xenial/ca-certificates) with recent update date 2015-04-26. 

The endpoint for package download is here: (http://archive.ubuntu.com/ubuntu/pool/main/c/ca-certificates/).

### CODE REVIEW

**Comment by lziest** [comment]
cool, would you mind adding one entry in https://github.com/cloudflare/cfssl_trust/blob/master/ca-bundle.crt.metadata as well? put 'weight' as 1, and we can adjust later.

**Comment by anniephan** [comment]
I notice that a majority (but not all) of the certs in ubuntu.pem use signature algorithm SHA1 with RSA. Should I just put these as the "hash_algo" and "key_algo" entries in the metadata file for Ubuntu?

**Comment by lziest** [comment]
that part is a little confusing. they should be 'latest_hash_algo_support' and 'latest_key_algo_support', so for ubuntu, I believe they are 'SHA2' and 'ECDSA256'

**Comment by anniephan** [comment]
Alright, added.

File: ca-bundle.crt.metadata [hunk 1]
```diff
@@ -109,4 +109,11 @@
 	"key_algo": "ECDSA256",
 	"keystore": "trusted_roots/kitkat.pem"
 }
+{
+	"name":"Ubuntu",
+	"weight": 1,
+	"latest_hash_algo_support": "SHA2",
```
Comment (lziest) [inline ca-bundle.crt.metadata]: sorry I confuse you what I mean is the key name originally should be named as that. But as right now we use "hash_algo" and "key_algo"

**Comment by lziest** [comment]
LGTM, please squash to one commit. Thanks!

File: ubuntu_update.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+// "tar xvf packageName" will create a directory without the ".tar.xz" extension
+// and will also replace all underscores "_" with dashes "-"
+var dirName string = strings.Replace(strings.TrimSuffix(packageName, ".tar.xz"), "_", "-", -1)
+
+func main() {
+	// Download the package to local directory
+	output, _ := os.Create(packageName)
+	defer output.Close()
+
+	res, _ := http.Get(updateURL + packageName)
```
Comment (dol) [inline ubuntu_update.go]: Please add at least a check sum check or the signature of the tar.gz with a hardcoded version of http://archive.ubuntu.com/ubuntu/pool/main/c/ca-certificates/ca-certificates_20150426ubuntu1.dsc .

Comment (lziest) [inline ubuntu_update.go]: Thanks @dol I would like to improve it with automatic verification with each package. I would also do it along with some package name parsing so we can always download the latest one.

Comment (lziest) [inline ubuntu_update.go]: I will raise a issue.

Comment (dol) [inline ubuntu_update.go]: @lziest Much appreciated. You find the latest source code of the package here: http://bazaar.launchpad.net/~ubuntu-branches/ubuntu/wily/ca-certificates/wily/files/head:/

**Pull request merged**